config/detekt.yml

```diff
@@ -82,7 +83,7 @@ formatting:
   MaximumLineLength:
     active: true
   ImportOrdering:
-    active: true
+    active: false
```

Comment:
with this `true`, i cannot order import in a way that makes detekt happy. detekt does not like how Square style formatter in intellij does import ordering :(